tests/python/unittest/test_tir_transform_lower_opaque_block.py

```diff
@@ -250,6 +250,34 @@ def transformed_strided_buffer_func(
             C[i0 * 4 + i1, j] = B[i1, j] * T.float32(2)
 
 
+@T.prim_func
+def compacted_symbolic_strided_buffer_func(a: T.handle) -> None:
+    n = T.int64()
+    A = T.match_buffer(a, (1, n, 10240), "float32")
+    for i, j, k in T.grid(((n + 63) // 64 * 4 + 7) // 8, 2, 160):
```

Comment:
Unrelated, the presence of this expression is kind of odd to me.  Assuming this example came from a TIR printout, I would have expected `((n + 63) // 64 * 4 + 7) // 8` to be simplified to the equivalent `(n + 127) // 128`.  The fact that it didn't simplify may indicate that I should take a look at the `CanonicalSimplifier`.